resources/interactive.mjs

```diff
@@ -196,18 +196,8 @@ function createUIForSuites(suites, onStep, onRunSuites) {
 }
 
 function startTest() {
-    if (params.suites.length > 0) {
-        if (!Suites.enable(params.suites)) {
```

Comment:
Eventually I guess we'll want to call `enable` with only the "default" tag if nothing is passed, right?

I actually prefer to keep the behavior how you have it here for now while we're still pre-release and working to get tests out of tentative.

Reply:
yes 👍